pkg/generated/openapi/openapi_test.go

```diff
@@ -23,12 +23,13 @@ import (
 
 	"github.com/go-openapi/spec"
 
+	generatedopenapi "k8s.io/apiextensions-apiserver/pkg/generated/openapi"
```

Comment:
this is another spec. This test here tests the kube-apiserver one which is not checked into the repository.